src/halium-lxc-exec.sh

```diff
@@ -22,5 +22,11 @@ found_path=$(whereis -b -B ${ANDROID_SEARCH_PATH} -f ${TARGET_BINARY} | head -n
 # Unset eventual LD_PRELOAD
 unset LD_PRELOAD
 
-# Finally execute
-exec /usr/bin/lxc-attach -n ${LXC_CONTAINER_NAME} -- ${found_path/${LXC_CONTAINER_PATH}/} ${@}
+# Some devices are broken: last argument is removed by lxc-attach
+# Work around this issue until someone find the reason
+check_broken_device=$(/usr/bin/lxc-attach -n ${LXC_CONTAINER_NAME} -- /bin/echo working)
```

Comment:
what do you think about some flag file in /var/lib/droidian/lxc_attach_workaround ? Then we can ship it on the adaptation (rather than checking every time)

Reply:
Should do the job too :)

Reply:
Done